R/LScommon.R

```diff
@@ -36,6 +36,12 @@ gettextf <- function(fmt, ..., domain = NULL)  {
               "The parameter '%1$s' for model/hypothesis '%2$s' must be positive.",
               gsub("par", "", names(models[[p]])[i]),
               models[[p]][["name"]]))
+        } else if (type %in% c("binEst", "binTest") && names(models[[p]])[i] %in% c("truncationLower", "truncationLower") && (models[[p]][[i]] < 0 || models[[p]][[i]] > 1)) {
+          .quitAnalysis(
+            gettextf(
+              "The truncation range for model/hypothesis '%2$s' must be within [0, 1] interval.",
```

Comment:
```suggestion
              "The truncation range for model/hypothesis '%1$s' must be within [0, 1] interval.",
```

Reply:
oh thanks, missed that when removing the other arguments